src/components/delete-button/delete-button.jsx

```diff
@@ -0,0 +1,34 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+
+import styles from './delete-button.css';
+import deleteIcon from './icon--delete.svg';
+
+const DeleteButton = props => (
+    <div
+        aria-label="Delete"
+        className={classNames(
+            styles.deleteButton,
+            props.className
+        )}
+        role="button"
+        tabIndex="0"
```

Comment:
Eventually this should probably be a prop with a default value of `0` so it can be changed.  But for now I think if they're all the same, the browser does the right thing anyway.